src/main/webapp/placeGuides/ListPlaceGuideDisplayer.js

```diff
@@ -2,75 +2,132 @@
  * This class is responsible for representing
  * the placeGuides on a scrollable list.
  */
-class ListPlaceGuideDisplayer {
-  constructor() {
-    this._placeGuidesOnList = {};
-  }
 
-  update(placeGuides) {
-    this.removePreviousPlaceGuidesFromList(placeGuides);
-    this.addNewPlaceGuidesToList(placeGuides);
-  }
+class ListPlaceGuideDisplayer {
 
-  removeFromList(placeGuideId) {
+  static QUERY = {
+    "ALL_PUBLIC_IN_MAP_AREA": {
+      listTitle: "Discover Guides",
+      listSubTitle: "in selected map area"
+    },
+    "CREATED_ALL_IN_MAP_AREA": {
+      listTitle: "My Guides",
+      listSubTitle: "in selected map area"
+    },
+    "BOOKMARKED": {
+      listTitle: "Bookmarked Guides",
+      listSubTitle: ""    }
+  };
```

Comment:
I think the ListPlaceGuideDisplayer should not work based on query types, but rather based on the page. We may have different subtitle on pages on which the same query is applied and vice-versa.
Please modify this enum.

Reply:
modified enum